### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by Python1320**
Rewrote/redid a hook an alternative way

### CODE REVIEW

**Comment by oldmud0** [comment]
Uh, isn't `CurTime()` a relatively expensive operation?

**Comment by immibis** [comment]
I wouldn't expect CurTime to be expensive. But then, I haven't actually tested it.

File: lua/wire/server/wirelib.lua [hunk 1]
```diff
@@ -506,8 +505,8 @@ function WireLib.TriggerOutput(ent, oname, value, iter)
 
 	local output = ent.Outputs[oname]
 	if (output) and (value ~= output.Value or output.Type == "ARRAY" or output.Type == "TABLE") then
-		if (output.TriggerLimit <= 0) then return end
```
Comment (thegrb93) [inline lua/wire/server/wirelib.lua]: This return is supposed to block too many triggers, but you removed it and didn't put in place a different check. How is the triggering limited?

Comment (Python1320) [inline lua/wire/server/wirelib.lua]: Good find. That's what you get for writing things past midnight. I was writing this thing in frustration of profiler output. I've actually written this before but forgot to pull request or something: https://github.com/Python1320/wire/commit/3eed48de913d98ea08a02a936fa9d0be970aa03b

**Comment by Python1320** [comment]
CurTime sure is less expensive than iterating and writing to an array of hundreds of entries plus the CurTime is not called during checking time limit.

Due to https://github.com/wiremod/wire/pull/1021#discussion_r44858136 I'll close this and reopen a new PR with the older patch code.

**Comment by immibis** [comment]
@Python1320 You know you can just push more stuff to the same branch and it will update the PR right? No need to create a new one.

**Pull request closed without merging**